jina/enums.py

```diff
@@ -88,15 +88,16 @@ def _from_yaml(cls, constructor, node):
 
 class SchedulerType(BetterEnum):
     LOAD_BALANCE = 0  #: balance the workload between Peas, faster peas get more work
-    ROUND_ROBIN = 1  #: workload are scheduled round-robin manner to the peas, assuming all peas have uniform processing speed.
+    ROUND_ROBIN = 1  # : workload are scheduled round-robin manner to the peas, assuming all peas have uniform
+    # processing speed.
```

Comment:
```suggestion
    ROUND_ROBIN = 1  # : workload are scheduled round-robin manner to the peas, assuming all peas have uniform
    # processing speed.
``` 
`#: ` is inline comment